client/src/theme/tabs.js

```diff
@@ -23,11 +30,11 @@ export default {
       }
 
       > button[aria-expanded="true"] {
-        font-weight: 700;
-        border-left: 1px solid ${theme.global.colors.border.light};
-        border-top: 1px solid ${theme.global.colors.border.light};
-        border-right: 1px solid ${theme.global.colors.border.light};
-        border-bottom: 1px solid ${theme.global.colors.background.light};
+        border-bottom: 2px solid ${theme.global.colors.brand.light};
```

Comment:
Another way to pull colors from the theme without defining them as hex values would be to use the `normalizeColor` util. You can see it used here: https://github.com/AlexsLemonade/scpca-portal/blob/c285a5b6445be16b3d47a0b19317c98614771b46/client/src/theme/select.js#L28

I know that we use this style of grabbing colors elsewhere but ideally we shouldn't be converting colors to hex in order to use them this way.

```suggestion
        border-bottom: 2px solid ${normalizeColor('brand', props.theme)};
```

Reply:
Let's reorganize the theme files at some point (e.g., create a separate theme file for `colors`) and specify our preferred approach to pulling colors (using `normalizeColor` or dot notation), appreciate your tip, David.